app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
Bạn cũng có thể viết như này để clean hơn nhé 
```suggestion
    @user = User.find_by_id params[:id]
```